src/app/shapes/Triangle.ts

```diff
@@ -10,11 +10,16 @@ export class Triangle extends Shape {
     }
   
     public getLeft(): number{
-        return Math.min(this.instance.points()[0], this.instance.points()[2], this.instance.points()[4]);
+        return this.instance.x() + Math.min(this.instance.points()[0], this.instance.points()[2], this.instance.points()[4]);
     }
    
     public getTop() : number{
-        return Math.min(this.instance.points()[1], this.instance.points()[3], this.instance.points()[5]);
+        return this.instance.y() + Math.min(this.instance.points()[1], this.instance.points()[3], this.instance.points()[5]);
+    }
+
+    public move(x: number, y: number): void {
+        this.instance.x(x);
+        this.instance.y(y);
     }
    
```

Comment:
When you change x and y, does konva do the math to move the whole triangle?(approving with the assumption this was tested)